server/routes/server.go

```diff
@@ -111,6 +117,13 @@ func NewServer(file string) (server *Server, err error) {
 
 	// notify admins
 	go router.NotifyAdmins()
+	// periodic deployments
```

Comment:
I believe we need to refactor a little bit, routes is more than routes now,

I suggest:
- App struct (that has the logger, router, .. etc)
- Also it should have a method to start background workers e.g mailsender, periodic checks .. etc